canonical-web-and-design/juju.is
# Convert site to flask

## Done
Converted the current site to flask from Jekyll

## QA
- Run `./run clean` then switch to this branch
- Run `./run build` and check it works
- Run `./run` and see the site at http://0.0.0.0:8041/
- Run `./run test` and check that passes
- Finally, try out `./run clean` and see that works

Fixes https://github.com/canonical-web-and-design/juju.is/issues/3
